pyblish_qml/__init__.py

```diff
@@ -5,9 +5,9 @@
 )
 
 
-def show(parent=None, targets=[]):
+def show(parent=None, targets=[], modal=None):
```

Comment:
Iiiih while you're at it, could you remove the `[]` in targets?

Pass `None` and convert it to a list within function instead. Lists in signature arguments are mutable, which means if you modify the list, the next call the the function (without `targets=`) will behave differently. Very difficult to debug!